src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -184,6 +185,35 @@ class StudyService(
         private val GET_ORGANIZATION_ID = """
             SELECT ${ORGANIZATION_ID.name} FROM ${ORGANIZATION_STUDIES.name} WHERE ${STUDY_ID.name} = ? LIMIT 1 
         """.trimIndent()
+
+        /**
+         * get studies that belong to the provided organizationId where the
+         * current user has read access
+         * include list of all organizationIds each study is a part of.
+         * sort by most recently created
+         * 1. current user principal id
+         * 2. organization id
+        */
+        private val GET_ORG_STUDIES_SQL = """
+            SELECT ${STUDIES.name}.*, org_studies.${ORGANIZATION_IDS.name}
+            FROM ${STUDIES.name}
+                INNER JOIN ${PERMISSIONS.name}
+                ON ${STUDIES.name}.${STUDY_ID.name} = ANY(${PERMISSIONS.name}.${ACL_KEY.name})
```

Comment:
This won't matter for this purpose but noting it should be ```ACL_KEY.name[1]``` as the reason that it is an array is that so there is somewhat of hierarchy (only used by roles atm).